apps/api/src/domains/admin/feedback/feedback.os.service.ts

```
@@ -569,12 +587,12 @@ export class FeedbackOSService {
     } = dto;
 
     for (let i = 0; i < (queries?.length ?? 0); i++) {
-      if (queries?.[i].issueIds) {
+      if (queries?.[i].key === 'issueIds') {
         const feedbackIds = await this.issueIdsToFeedbackIds(
-          queries[i].issueIds as number[],
+          queries[i].value as number[],
         );
 
-        delete queries[i].issueIds;
+        delete queries[i].value;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)